src/object/notification.c

```diff
@@ -45,14 +45,6 @@ static inline void maybeDonateSchedContext(tcb_t *tcb, notification_t *ntfnPtr)
         sched_context_t *sc = SC_PTR(notification_ptr_get_ntfnSchedContext(ntfnPtr));
```

Comment:
Is this commit just a result of the previous commit 1b34e3bdaff816e6584bb795e09e7058416abd0d introducing cases where refill_unblock_check is called multiple times in a single kernel operation?

Reply:
This was things that verification had proved about the previous callsites but were not able to easily proved given the proof states at the new callsites. Not all of these may be necessary but proving that they aren't needed is more than we have bandwidth for.